docs/release-notes/NuGet-5.3.md

```diff
@@ -0,0 +1,90 @@
+
+## Summary: What's New in 5.3 preview 3
+
+* Enable SHA tracking and enforcement for Packages.Config scenarios - [#7281](https://github.com/NuGet/Home/issues/7281)
+
+* Enable server side deprecation of obsolete/legacy NuGet Packages - [#2867](https://github.com/NuGet/Home/issues/2867)
+
+* Package Icons should be embedded in Packages, instead of an external URL. - [#352](https://github.com/NuGet/Home/issues/352)
+
+### Issues fixed in this release
+
+**Bugs**
+
+* VS:  assemblies are fully ngen-ed not partially ngen-ed - [#8513](https://github.com/NuGet/Home/issues/8513)
+
+* Reduce memory usage (unsubscribe from events) - [#8471](https://github.com/NuGet/Home/issues/8471)
+
+* "Error_UnableToFindProjectInfo" message is not grammatically correct - [#8441](https://github.com/NuGet/Home/issues/8441)
+
+* NU1403 improvements - validate all packages, include the expected/actual sha values - [#8424](https://github.com/NuGet/Home/issues/8424)
+
+* Multiple enumeration in NuGetPackageManager.PreviewUpdatePackagesAsync - [#8401](https://github.com/NuGet/Home/issues/8401)
+
+* Revert "public -> internal" change in PluginProcess - [#8390](https://github.com/NuGet/Home/issues/8390)
+
+* IVsPackageSourceProvider.GetSources(…) has ill-defined exception behavior - [#8383](https://github.com/NuGet/Home/issues/8383)
+
+* Make PluginManager constructor public again - [#8379](https://github.com/NuGet/Home/issues/8379)
+
+* Metrics to track the refresh rate of the PM UI - [#8369](https://github.com/NuGet/Home/issues/8369)
+
+* Decrease the number of UI refreshes when installing through the Package Manager UI - [#8358](https://github.com/NuGet/Home/issues/8358)
+
+* Telemetry:  datetime values use culture-specific formats - [#8351](https://github.com/NuGet/Home/issues/8351)
+
+* Reduce UI refreshes in browse tab of Package Manager UI #6570 - [#8339](https://github.com/NuGet/Home/issues/8339)
+
+* [Test Failure] “Unable to parse config file” will prompt twice - [#8320](https://github.com/NuGet/Home/issues/8320)
+
+* Raise NU5037 error with good doc page that explains customer fixes (The package is missing the required nuspec file) - [#8291](https://github.com/NuGet/Home/issues/8291)
+
+* Locked-mode restore fails when a project's RuntimeIdentifier is changed - [#8260](https://github.com/NuGet/Home/issues/8260)
+
+* Make the Settings reading in VS lazy - [#8156](https://github.com/NuGet/Home/issues/8156)
+
+* Regression in 'Nuget sources add' causes "The ':' character, hexadecimal value 0x3A, cannot be included in a name" errors - [#7948](https://github.com/NuGet/Home/issues/7948)
+
+* NuGet plugin credential providers - hide the process window - [#7511](https://github.com/NuGet/Home/issues/7511)
+
+* Enforce PackagePathResolver is an absolute path - [#7349](https://github.com/NuGet/Home/issues/7349)
+
+* Reduce UI refreshes in install and update tabs of Package Manager UI - [#6570](https://github.com/NuGet/Home/issues/6570)
+
+**DCR:**
```

Comment:
Why did we merge the DCR and Feature stuff? :D

Reply:
its not merged. I moved the feature to the top as part of the summary.